### ISSUE TITLE
Missing semantic differentiation between status and heading

### ISSUE THREAD

**Issue by joe-replin** [adaptlearning/adapt-contrib-core#151]
## Bug Description:
In the table of contents, the status text "Complete" and "Incomplete" is included in the lesson title without semantic differentiation. This can render understanding of the lesson title unclear for screen reader users.

## Steps to Reproduce:
Using a screen reader, open the course prototype.
Navigate through the course titles and note the description of each course.

## Actual Behavior:
```
<div id="co-100-heading" class="js-heading-inner" role="heading" aria-level="2">
  <span class="aria-label">
  Incomplete Standard experience
  </span>
</div>
```

## Expected Behavior:
Update the labels of the lessons to provide clear separation between the lesson title and its status. The text "complete" by itself at the beginning of the heading can still make the purpose of the heading misleading without additional context. Consider moving status text to the end of the heading with context as to what the status represents (ex"[Course Title], course completed")

**Comment by joe-replin** [comment]
Reopening this after further conversations on this to finalize plans. Status indicators in the aria-label should come first. Additionally, without context as to what is the status is reporting on can cause confusion as to the purpose of the status indicators. It was agreed the best path forward was to present all aria-headers in the following format:

[context (if applicable)] [status]. [title]
